l10n_it_account_stamp/models/company.py

```diff
@@ -21,15 +21,3 @@ class AccountConfigSettings(models.TransientModel):
         help="Product used as Tax Stamp in customer invoices.",
         readonly=False
         )
-
-    @api.onchange('company_id')
-    def onchange_company_id(self):
```

Comment:
Come mai `onchange_company_id` rimosso?

Reply:
serve? c'è il related

Reply:
> serve?

Penso di no.

Allora idealmente sarebbe da mettere in un commit dedicato o almeno scrivere in questo il motivo della rimozione; perchè può non essere chiaro il motivo della modifica

Reply:
> > serve?
> 
> Penso di no.
> 
> Allora idealmente sarebbe da mettere in un commit dedicato o almeno scrivere in questo il motivo della rimozione; perchè può non essere chiaro il motivo della modifica

Tra l'altro ho visto che quell'onchange solleva un'eccezione quando, su un db nuovo, vado su _Invoicing→Configuration→Settings_

l10n_it_account_stamp/models/company.py", line 27, in onchange_company_id
    res = super(AccountConfigSettings, self).onchange_company_id()
AttributeError: 'super' object has no attribute 'onchange_company_id'